x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/enrollment/ClientEnrollmentAction.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.core.security.action.enrollment;
+
+import org.elasticsearch.action.ActionType;
+
+public final class ClientEnrollmentAction extends ActionType<ClientEnrollmentResponse> {
+
+    public static final String NAME = "cluster:admin/xpack/enroll/client";
```

Comment:
Yeap, not only for consistency but also so that the privilege matchers apply in the same way. Given where that discussion currently is, I'll change this to `cluster:admin/xpack/security/enrollment/enroll/client`